README.md

```diff
@@ -220,7 +220,7 @@ See [customize options] and [manage connection profiles] for more details.
     "mssql.logDebugInfo": false,
     "mssql.saveAsCsv.includeHeaders": true,
     "mssql.saveAsCsv.delimiter": ",",
-    "mssql.saveAsCsv.lineSeperator": null,
+    "mssql.saveAsCsv.lineSeparator": null,
```

Comment:
Fix the user-facing portion of https://github.com/Microsoft/vscode-mssql/issues/1198.